--- /usr/share/festival/voices/us/nitech_us_rms_arctic_hts/festvox/nitech_us_rms_arctic_hts.scm.old	2012-07-17 12:01:35.179290044 +0700
+++ /usr/share/festival/voices/us/nitech_us_rms_arctic_hts/festvox/nitech_us_rms_arctic_hts.scm	2012-07-17 12:03:10.445294146 +0700
@@ -68,7 +68,7 @@
 (set! hts_feats_list
       (load (path-append hts_data_dir "feat.list") t))
 
-(require 'hts)
+(require 'hts21compat)
 (require_module 'hts_engine)
 
 ;;; Voice specific parameter are defined in each of the following
@@ -162,7 +162,7 @@
   ;; Waveform synthesis model: hts
   (set! hts_engine_params nitech_us_rms_arctic_hts::hts_engine_params)
   (set! hts_feats_list nitech_us_rms_arctic_hts::hts_feats_list)
-  (Parameter.set 'Synth_Method 'HTS)
+  (Parameter.set 'Synth_Method 'HTS21)
 
   ;; This is where you can modify power (and sampling rate) if desired
   (set! after_synth_hooks nil)
